web/src/invite.ts

```diff
@@ -288,6 +290,18 @@ function set_streams_to_join_list_visibility(): void {
     }
 }
 
+function generate_invite_tips_data(): Record<string, boolean> {
+    const {realm_description, realm_icon_source, custom_profile_fields} = page_params;
+
+    return {
+        is_description:
+            realm_description !== "" &&
+            !/^Organization imported from [A-Za-z]+[!.]$/.test(realm_description),
+        is_profile_pic: realm_icon_source !== "G",
+        is_custom_profile: custom_profile_fields.length > 0,
```

Comment:
I think these variables can have better names like - `is_realm_description_set` or maybe `is_realm_description_configured`,  `is_realm_icon_user_uploaded` or `is_realm_profile_picture_user_uploaded`, `realm_has_custom_profile_fields`.

Basically, it would be better to use `custom_profile_fields` instead of `custom_profile` and `realm_icon` instead of `profile_pic` and be a little bit verbose to explain the use of variable.

Reply:
I have renamed the variables to: `realm_has_description`, `realm_has_user_set_icon` and `realm_has_custom_profile_fields` to balance the variable length vs verbosity. Does this sound good to you?